package.json

```diff
@@ -25,9 +25,9 @@
     "artillery-report": "artillery report ./reports/all-results.json",
     "snyk": "snyk",
     "precommit": "yarn run git-hook",
-    "prepush": "yarn run git-hook && yarn run snyk -- test",
+    "prepush": "yarn run git-hook",
     "postrewrite": "yarn run git-hook",
-    "travis": "yarn run git-hook && yarn run upload-coverage-coveralls",
+    "travis": "yarn run git-hook && yarn run upload-coverage-coveralls && yarn run snyk -- test --org=nhsuk ignore --id=npm:moment:20161019 && yarn run snyk -- monitor --org=nhsuk",
```

Comment:
Happy /w the answers received offline about snyk ignoring the patches and the necessity of this approach.